kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm-ui/src/build/revapi-config.json

```diff
@@ -22,11 +22,42 @@
       }
     }
   },
-
   "ignores": {
     "revapi": {
-    "_comment": "Changes between 7.48.0.Final and the current branch. These changes are desired and thus ignored.",
-    "ignore": [
+      "_comment": "Changes between 7.63.0.Final and the current branch. These changes are desired and thus ignored.",
```

Comment:
@bxf12315 it seems the reference version is 7.52.0.Final instead of 7.63.0.Final